### ISSUE TITLE
Investigate removing `import ... as` usage

### ISSUE THREAD

**Issue by alanking** [irods/python-irodsclient#646]
Renaming imported modules and module members can be confusing when reading code. We should consider removing these in favor of the actual module names in order to prevent confusion and - in some cases - collisions.

For example:
https://github.com/irods/python-irodsclient/blob/f2394e01fa9194741776258fed771c0842f07192/irods/message/__init__.py#L9-L12

This can be addressed in a patch release. This can be closed as resolved/invalid if we think it's unnecessary or detrimental.

**Comment by d-w-moore** [comment]
Is `import ... as` considered to be a deprecated practice in Python?

**Comment by alanking** [comment]
No. There was a Codacy issue raised against PR #640 which says "Redefining built-in 'property'" with respect to an `import ... as` statement:
```python
from . import property_types as property
```
Rather than make an issue specific to this, I thought it may be worth our while to investigate removing this practice or at least taking a pass over existing usage to see whether it can be improved.

I personally find this practice hard to follow (I liken it to C++ `using` or `namespace` directives, but worse), but I certainly think it's fine to keep it. Just created the issue while I was thinking about it.

This issue can address any/all `import ... as`, just the one redefining `property`, or none at all.

**Comment by d-w-moore** [comment]
@alanking ok.  I think we'll just have to rename the `irods/message/property.py` module then.  It's a potentially breaking change, so if we're going to do that we might as well do it with` v3.0.0`.

**Comment by korydraughn** [comment]
Yep, this is the time.

**Comment by d-w-moore** [comment]
> Yep, this is the time.

Sounds good to me.

**Comment by alanking** [comment]
The change which originally inspired this issue originated in #640, so, never actually existed. If we want to do a sweep on the other uses of `import ... as`, we can certainly do that, but I wouldn't consider a blocker for 3.0.0.

We will either want to move this out of the milestone or close as resolved/invalid.

**Comment by korydraughn** [comment]
So https://github.com/irods/python-irodsclient/issues/646#issuecomment-2429237419 isn't real anymore?

I don't see anything in the codacy report about that for the merged PR.

**Comment by alanking** [comment]
Correct. The line originated there and then was removed.

**Comment by korydraughn** [comment]
Good. I think using the true module name is a good thing.

Let's keep this open and aim to restore use of the true module name.

As @alanking said, this isn't a blocker for 3.0.0. We can punt this issue as needed.

**Comment by d-w-moore** [comment]
I removed this line `from . import property_types as property` because in its context in irods/message/__init__.py it's only there to provide a stopgap against the change being a breaking change for anyone that might've relied on the existence of the irods.message.property namespace alias.  It was sensitive to order (needing to come after all the `@property` directives) in that regard -- particularly being there without a comment -- so I removed it.

But as far as **import A as B** in general usage, this syntax is straightforward.  It basically imports a module namespace and renames it at the same time. I do not think it's confusing to most Python programmers or generally in need of removal overall.  Tbh I think most people are far more confused by relative imports -- and those are basically here to stay, a necessary evil you might say.)
